library/analytics/src/main/java/org/cru/godtools/analytics/firebase/FirebaseAnalyticsService.kt

```diff
@@ -18,6 +16,7 @@ import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.flow.onEach
 import org.ccci.gto.android.common.okta.oidc.OktaUserProfileProvider
 import org.ccci.gto.android.common.okta.oidc.net.response.ssoGuid
+import org.ccci.gto.android.common.compat.util.LocaleCompat
```

Comment:
ktlint is complaining that the import isn't properly sorted. you should be able to select the imports and use the Android Studio re-formatting to automatically fix this

Reply:
Okay I'll go ahead and do that.